guests_app/src/ViewModel/AuthenticateViewModel.ts

```diff
@@ -0,0 +1,13 @@
+import Requests from '../Networking/requests';
+
+export default class AuthenticateViewModel {
+	private requests: Requests;
+
+	constructor(requests: Requests) {
+		this.requests = requests;
+	}
+
+	login(phone_number: String, password: String) {
+		return this.requests.login(phone_number, password);
```

Comment:
After logging in the server should return a token that needs to be saved and sent with your other requests